SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+We aim to keep Timescale safe for everyone. 
+Publicly disclosing bugs in a public forum puts everyone in the Timescale community at risk.
+
+## Supported Versions
```

Comment:
Is there another place we call out supported versions? It would be best to link out to that than have this become unmaintained.

Reply:
See my comment bellow.